builtin/types/transfer_gateway/transfer_gateway.proto

```diff
@@ -317,6 +317,17 @@ message TransferGatewayConfirmWithdrawalReceiptRequest {
     bytes withdrawal_hash = 3; // TODO: figure out if there's any point passing this around
 }
 
+message TransferGatewayConfirmWithdrawalReceiptRequestV2 {
+    // DAppChain address of the entity attempting to make the withdrawal
+    Address token_owner = 1;
+    // 66-byte hash of the withdrawal hash
```

Comment:
update comment to make it clear that the oracle_signature here represents a serialized array of 65 byte signatures (they aren't 66 in the distributed gateway case)